frontend/src/components/Login/Password.jsx

```diff
@@ -44,14 +44,16 @@ class Password extends React.Component {
   }
   render() {
     const { errorText } = this.state
+    const { validatingError } = this.props
+    const foregroundErrorText = errorText || validatingError
```

Comment:
Because I thought it would be the most appropriate name for the value that is more important at the moment.